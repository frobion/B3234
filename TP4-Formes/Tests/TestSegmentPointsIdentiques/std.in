S Segment1 1 2 12 13
S Segment2 1 2 1 2
LIST
EXIT

