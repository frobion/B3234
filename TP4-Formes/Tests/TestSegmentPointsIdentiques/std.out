OK
ERR
# Points identiques
S Segment1 1 2 12 13

